### ISSUE TITLE
Update from 3.4.3.0/3.43 doesn't work.

### ISSUE THREAD

**Issue by Athanasius** [EDCD/EDMarketConnector#804]
This was initially reported by a user on Discord.  I've just tested it myself and see the same issue.

Using the last version that @Marginal built and released, 3.4.3.0/3.43, WinSparkle appears to crash out shortly after offering an update.

1. Uninstall newer version.
2. Install [3.4.3.0](https://github.com/EDCD/EDMarketConnector/releases/tag/rel-343)
3. The installation process will have run the program, so now perform a manual update check via `Help` > `Check for updates`.
4. See the WinSparkle "there's an update!" window popup, but
    1. The pane that should display the changelog is blank.
    2. 2-3 seconds later it closes, and takes the EDMarketConnector.exe process with it.

This doesn't happen with the 3.4.4.0 version that I first built and released.

**Comment by Athanasius** [comment]
I've sent an email to @Marginal asking for them to change the redirect on the appcast file to point at a URL I directly control.  That way I can experiment to see if changing the content of the appcast file can fix this.

**Comment by Athanasius** [comment]
I've just performed an experiment with the 'updates' file. I'm sure most users don't read the changelog that's presented when the automatic updater offers a new version, and certainly not all the change logs for all the previous versions back to 3.0.

So, I removed everything except the most recent changelogs (the warning in the MacOS section about no longer doing builds, and all but 4.1.6 for Windows). Removed EDMC 4.1.6, installed 3.4.3.0 again, and ...

It only bloody worked. 3.4.3.0 detected the upgrade to 4.1.6, offered it to me, didn't crash when showing the changelog, and then correctly performed the upgrade. So hopefully we'll now reduce the holdouts on *all* pre-4.0.0.0 versions :) .
